analysis/vaccine_coverage_calc.R

```diff
@@ -0,0 +1,99 @@
+## ###########################################################
+
+##  This script:
+## - Imports data of the three waves
+## - Calculates vaccine coverage for each subgroup
+
+## linda.nab@thedatalab.com - 20220608
+## ###########################################################
+
+# Load libraries & functions ---
+library(here)
+library(readr)
+library(purrr)
+library(dplyr)
+library(jsonlite)
+## Load json file listing demographics and comorbidities
+config <- fromJSON(here("analysis", "config.json"))
+## Create vector containing the demographics and comorbidities
+subgroups_vctr <- c("agegroup", "sex", config$demographics, config$comorbidities)
+
+# Import data extracts of waves  ---
+input_files_processed <-
+  Sys.glob(here("output", "processed", "input_wave*.rds"))
+data_processed <- 
+  map(.x = input_files_processed,
+      .f = ~ readRDS(.x))
+names(data_processed) <- c("wave1", "wave2", "wave3")
+
+# Functions ---
+## 'calc_n_vax' creates a summary table for a specific subgroup with columns:
+## ssubgroup level n_vax_1 n_vax_2 n_vax_3 n cov_1 cov_2 cov_3
```

Comment:
Minor typo here but it's in the comments so won't affect the code itself

Reply:
Thank you. Fixed!